packages/api/src/services/stats/lsp.ts

```diff
@@ -95,16 +108,34 @@ export default (config: Config, appState: Dependencies) => {
     // normally you would upsert an lsp address where "global" is, but we are going to use a custom value to represent tvl across all addresses
     return stats[currency].latest.tvl.upsertGlobal(update);
   }
+  // tvm is just based on TVL for LSPS.
+  async function updateGlobalTvm() {
+    const value = await queries.totalTvm(currency);
+    const update = {
+      value,
+      timestamp: nowS(),
+    };
+    // normally you would upsert an lsp address where "global" is, but we are going to use a custom value to represent tvl across all addresses
+    return stats[currency].latest.tvm.upsertGlobal(update);
+  }
   async function update() {
     const addresses = Array.from(registeredLsps.values());
     await updateAllTvl(addresses).then((results) => {
       results.forEach((result) => {
-        if (result.status === "rejected") console.error("Error updating tvl: " + result.reason.message);
+        if (result.status === "rejected") console.error("Error updating TVL: " + result.reason.message);
```

Comment:
nit: should we be printing the error stack on these rather than just the message?

Reply:
it really depends. the only reason i didnt do this is because there would just be a ton of logs.  There are addresses that consistently fail when requesting data from price feeds and therefore they fail when calculating stats. Without a way to blacklist or silence certain addresses it ends up being really noisy without the stack trace.  here is a small sample, you could imagine this with stack traces for each line 
```